heron/api/src/java/com/twitter/heron/dsl/Streamlet.java

```diff
@@ -85,7 +85,7 @@
    * flattening the result
    * @param flatMapFn The FlatMap Function that should be applied to each element
   */
-  <T> Streamlet<T> flatMap(SerializableFunction<? super R, Iterable<? extends T>> flatMapFn);
+  <T> Streamlet<T> flatMap(SerializableFunction<? super R, ? extends Iterable<T>> flatMapFn);
```

Comment:
Would we rather have SerializableFunction<? super R, ? extends Iterable<? extends T>> flatMapFn

Reply:
What does that really buy us though?

Reply:
Ya we can change it to that